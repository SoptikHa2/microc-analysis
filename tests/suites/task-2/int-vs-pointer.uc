main() {
    var x;
    x = alloc 17;
    x = 41;
    return x + 1;
}