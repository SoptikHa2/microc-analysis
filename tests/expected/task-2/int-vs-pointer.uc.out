---
test: type suites/task-2/int-vs-pointer.uc
---
suites/task-2/int-vs-pointer.uc:4:9: type error: Cannot unify '⟦41[4:9]⟧' and 'int' (with canonicals '↑⟦17[3:15]⟧' and 'int')
      x = 41;
          ^


exit: 1
